src/components/NcAppNavigationItem/NcAppNavigationItem.vue

```diff
@@ -859,20 +903,21 @@ export default {
 }
 
 /* Makes the icon of the collapsible element disappear
-*  When hovering on the root element */
+ * When hovering on the root element
+ */
 .app-navigation-entry--collapsible {
-	//shows the triangle button
+	// hides the triangle button
 	.icon-collapse {
-		visibility: hidden;
+		display: none;
```

Comment:
@raimund-schluessler can the above be addressed by simply setting this to display:initial?